### PR TITLE
Replace Glyphicons with Font Awesome icons

### LINKED ISSUE #5292: Migrate away from Glyphicons
### Descriptive summary

Hyrax will likely need to transition to a different icon font, like [Fork Awesome](https://forkaweso.me/Fork-Awesome/) or [Octicons](https://octicons.github.com/), as Glyphicons is a non‐free icon set and dropped from Bootstrap 4.

### Rationale

Bootstrap 4 support.

### Related work

See #5276, projectblacklight/blacklight#1585, geoblacklight/geoblacklight#840.

### PR DESCRIPTION
**Pull request by adamjarling**
Fixes #5292 

This PR will replace references to Bootstrap 3 Glyphicons with Font Awesome for icon usage in the app.

Validating the PR basically would be just cruising through the app and noticing icons show up.   If any obvious icons have shifted or are not present, feel free to make note.

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Review by mathewjordan — APPROVED** [review]
I don't see any missed ones but from the interface I think this is good. What a process to convert these icons.

File: config/locales/hyrax.es.yml [hunk 1]
```diff
@@ -1031,21 +1031,21 @@ es:
           view_collection: Ver colección
           work_confirmation: El borrado de un trabajo de %{application_name} es permanente. Presione OK para borrar este trabajo de %{application_name}, o Cancelar para cancelar esta operación
         collection_list:
-          description: 'Descripción:'
-          edit_access: 'Editar Acceso:'
-          groups: 'Grupos:'
+          description: "Descripción:"
+          edit_access: "Editar Acceso:"
+          groups: "Grupos:"
```
[the remark this answers is not in the document]
Comment (adamjarling) [reply config/locales/hyrax.es.yml:1036]: Ah, yea probably auto formatting.  Looks like it's just in the internationalization `.yml` files.   I thought I was being good about saving w/o formatting but guess this slipped by with some multi instance find and replace.  @marrus-sh Think it's worth reverting these back to the original single quote format, or does it really matter?

**Review by Dananji — APPROVED** [review]
Looks good to me :tada:

File: config/locales/hyrax.es.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (marrus-sh) [reply config/locales/hyrax.es.yml:1036]: i’d probably revert, just to keep irrelevant commits out of the `git blame` for those files to the extent possible. i don’t think the actual single/double quote distinction makes much of a difference, i just want to keep the commit history nice :)

[the remark this answers is not in the document]
Comment (adamjarling) [reply config/locales/hyrax.es.yml]: @marrus-sh single/double quotes reverted in the affected local `.yml` files.

**Pull request merged**